src/test/java/api/requests/PickSlipsTests.java

```diff
@@ -0,0 +1,398 @@
+package api.requests;
+
+import static java.net.HttpURLConnection.HTTP_OK;
+import static org.hamcrest.core.Is.is;
+import static org.hamcrest.junit.MatcherAssert.assertThat;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+
+import java.net.MalformedURLException;
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+import java.util.UUID;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import api.support.APITests;
+import api.support.builders.RequestBuilder;
+import api.support.http.InventoryItemResource;
+import api.support.http.ResourceClient;
+import io.vertx.core.json.JsonArray;
+import io.vertx.core.json.JsonObject;
+import org.apache.commons.lang3.StringUtils;
+import org.folio.circulation.domain.RequestType;
+import org.folio.circulation.support.http.client.IndividualResource;
+import org.junit.Assert;
+import org.junit.Test;
+
+import org.folio.circulation.domain.RequestStatus;
+import org.folio.circulation.support.http.client.Response;
+
+public class PickSlipsTests extends APITests {
+
+  private static final int UUID_BATCH_SIZE = 50;
```

Comment:
If this is only used in a single test, I think it would be more readable if it were defined in that test, rather than having to navigate to the definition here

Reply:
done